---
title: JBC-编制光合作用蛋白质目录 揭秘生物过程
link: https://lufuhao.wordpress.com/2011/06/21/jbc%ef%bc%9a%e7%bc%96%e5%88%b6%e5%85%89%e5%90%88%e4%bd%9c%e7%94%a8%e8%9b%8b%e7%99%bd%e8%b4%a8%e7%9b%ae%e5%bd%95-%e6%8f%ad%e7%a7%98%e7%94%9f%e7%89%a9%e8%bf%87%e7%a8%8b/
post_id: 635
created: 2011/06/21 17:30:04
created_gmt: 2011/06/21 08:30:04
comment_status: open
post_name: JBC-编制光合作用蛋白质目录 揭秘生物过程
status: publish
post_type: post
---

# JBC：编制光合作用蛋白质目录 揭秘生物过程

据美国物理学家组织网近日报道，美国卡内基学院、加利福尼亚大学洛杉矶分校与美国能源部联合研究院利用先进的计算机工具，分析了28种植物中与光合作用相关的基因组，编制出与光合作用有关的597个编码基因蛋白的详细目录，从而可更好地从基因学角度研究支撑植物生理与生态的各种生物过程。研究论文发表在最新一期《生物化学杂志》上。 

这597个来自植物和绿藻基因组的编码蛋白，称为GreenCut蛋白质，是光合生物特有的蛋白。其中286个是当前已知的功能蛋白，剩下的311个尚无法与特定的生物过程联系起来。 

叶绿体是进行光合作用的工作间，有52%的GreenCut蛋白质位于叶绿体上。目前人们普遍认为，叶绿体是从一种能进行光合作用的单细胞细菌—— 藻青菌进化而来。大约15亿年前，藻青菌被更加复杂的、不能进行光合作用的细胞所吞噬，两种生物之间形成了最早的共生关系。在进化过程中，藻青菌将它的大部分基因信息转移给了宿主生物的细胞核，丧失了独立生存能力。 

“这种基因减退的藻青菌是叶绿体的基础，却保持了它的光合作用能力和某些基本的代谢功能，如合成氨基酸和脂肪。叶绿体中发生的这些过程，也必须和其他代谢过程紧密结合在一起。”卡内基学院研究员格鲁斯曼解释说。 

最近发现的证据表明，并非所有的GreenCut蛋白质都在叶绿体上，许多参与光合作用的GreenCut蛋白对于植物其他功能作用也起着关键作用，可能涉及新陈代谢调控、DNA转录控制、线粒体产生能量、过氧物酶体（能净化室内空气）等细胞器正常功能的发挥。 

进一步扩展研究范围之后，研究人员还发现，在古老的藻青菌、红藻及硅藻等其他单细胞藻类中，也保留了多种GreenCut蛋白质。格鲁斯曼表示，比较多种生物中所含有的GreenCut蛋白质，有助于进一步揭开这些蛋白质在光合细胞中的作用，研究叶绿体的进化过程，以及在不同条件下，光合细胞是如何调整变化以适应生存环境的。 

 

**The GreenCut2 Resource, a Phylogenomically Derived Inventory of Proteins Specific to the Plant Lineage**

Steven J. Karpowicz, Simon E. Prochnik, Arthur R. Grossman and Sabeeha S. Merchant

The Journal of Biological Chemistry, DOI:[10.1074/jbc.M111.233734](http://doi.org/10.1074/jbc.M111.233734)

The plastid is a defining structure of photosynthetic eukaryotes and houses many plant-specific processes, including the light reactions, carbon fixation, pigment synthesis, and other primary metabolic processes. Identifying proteins associated with catalytic, structural, and regulatory functions that are unique to plastid-containing organisms is necessary to fully define the scope of plant biochemistry. Here, we performed phylogenomics on 20 genomes to compile a new inventory of 597 nucleus-encoded proteins conserved in plants and green algae but not in non-photosynthetic organisms. 286 of these proteins are of known function, whereas 311 are not characterized. This inventory was validated as applicable and relevant to diverse photosynthetic eukaryotes using an additional eight genomes from distantly related plants (including Micromonas, Selaginella, and soybean). Manual curation of the known proteins in the inventory established its importance to plastid biochemistry. To predict functions for the 52% of proteins of unknown function, we used sequence motifs, subcellular localization, co-expression analysis, and RNA abundance data. We demonstrate that 18% of the proteins in the inventory have functions outside the plastid and/or beyond green tissues. Although 32% of proteins in the inventory have homologs in all cyanobacteria, unexpectedly, 30% are eukaryote-specific. Finally, 8% of the proteins of unknown function share no similarity to any characterized protein and are plant lineage-specific. We present this annotated inventory of 597 proteins as a resource for functional analyses of plant-specific biochemistry.
