title: 启动子分析数据库
link: https://lufuhao.wordpress.com/2010/08/17/%e5%90%af%e5%8a%a8%e5%ad%90%e5%88%86%e6%9e%90%e6%95%b0%e6%8d%ae%e5%ba%93/
author: lufuhao
description: 
post_id: 45
created: 2010/08/17 13:01:08
created_gmt: 2010/08/17 13:01:08
comment_status: open
post_name: %e5%90%af%e5%8a%a8%e5%ad%90%e5%88%86%e6%9e%90%e6%95%b0%e6%8d%ae%e5%ba%93
status: publish
post_type: post

# 启动子分析数据库

Higo K, Ugawa Y, Iwanmoto M, Korenaga T. Plant cis-acting regulatiory DNA elements (PLACE) database: 1999. **Nucleic Acids Res**, 1999, 27: 297-300 

http://www.dna.affrc.go.jp 

Lescot M, Dehais P, Thijs G, Marchal K, Moreau Y, Van de Peer Y, Rouze P, Rombauts S. PlantCARE, a database of plant cis-acting regulatory elements and a portal to tools for in silico analysis of promoter sequences. **Nucleic Acids Res**, 2002, 30: 325-327 

http://sphinx.rug.ac.be:8080/PlantCARE 

Frith MC, Hansen U, Weng Z. Detection of cis-element clusters in higher eukaryotic DNA. **Bioinformatics**, 2001, 17: 878-889 

http://zlab.bu.edu 

Bucher P. Weight matrix descriptions of four eukaryotic RNA polymerase II promoter elements derived from 502 unrelated promoter sequences. **J Mol Biol,** 1990, 212: 563-578 

Joshi CP. An inspection of the domain between putative TATA box and translation start site in 79 plant genes.** Nucl Acids Res,** 1987, 15: 6643-6653 

Devereux J, Haeberli P, Smithies O. A comprehensive set of sequence analysis programs for the VAX. **Nucleic Acids Res**, 1984, 12: 195-387 

Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. **J Mol Biol**, 1990, 215: 403-410 

Emanuelsson O, Nielsen H, Brunak S, Heijne G. Predicting subcellular localization of proteins based on their N-terminal amino acid sequence. **J Mol Biol**, 2000, 300: 1005-1016 

Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG. The ClustalX windows interface: Flexible strategies for multiple sequence alignment aided by quality analysis tools. **Nucl Acids Res,** 1997, 24: 4876-4882 

Quandt K, Frech K, Karas H, Wingender E, Werner T. MatInd and MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data. **Nucl Acids Res**, 1995, 23: 4878-4884 

Werner T. Computer-assisted analysis of transcription control regions: MatInspector and other programs. **Methods Mol Biol,** 2002, 132: 337-349 

Zuker M. Mfold web server for nucleic acid folding and hybridization prediction. **Nucl Acids Res**, 2003, 31: 3406-3415 

Alignment of Max Promoter Regions：Sequence comparisons were made using the CLUSTAL W program of the Sequence Interpretation Tools section of GenomeNet at http://www.genome.ad.jp/.