title: PLoS ONE：六种木本竹子叶绿体基因组全序列测定完成
link: https://lufuhao.wordpress.com/2011/06/07/plos-one%ef%bc%9a%e5%85%ad%e7%a7%8d%e6%9c%a8%e6%9c%ac%e7%ab%b9%e5%ad%90%e5%8f%b6%e7%bb%bf%e4%bd%93%e5%9f%ba%e5%9b%a0%e7%bb%84%e5%85%a8%e5%ba%8f%e5%88%97%e6%b5%8b%e5%ae%9a%e5%ae%8c%e6%88%90/
author: lufuhao
description: 
post_id: 619
created: 2011/06/07 20:43:48
created_gmt: 2011/06/07 11:43:48
comment_status: open
post_name: plos-one%ef%bc%9a%e5%85%ad%e7%a7%8d%e6%9c%a8%e6%9c%ac%e7%ab%b9%e5%ad%90%e5%8f%b6%e7%bb%bf%e4%bd%93%e5%9f%ba%e5%9b%a0%e7%bb%84%e5%85%a8%e5%ba%8f%e5%88%97%e6%b5%8b%e5%ae%9a%e5%ae%8c%e6%88%90
status: publish
post_type: post

# PLoS ONE：六种木本竹子叶绿体基因组全序列测定完成

来自中国科学院昆明植物研究所等处的研究人员利用新一代测序技术Illumina完成了六种木本竹子叶绿体基因组全序列的测定，为分析木本竹子的机制机理以及系统发育学研究提供了参考。这项研究获得了中国科学院知识创新工程（KSCX-YW-N-029）、国家自然科学基金项目（30990244）和云南省创新团队的资助，研究成果公布在《公共科学图书馆—综合》（PLoS ONE）杂志上。

![2011060710345782](http://lufuhao.files.wordpress.com/2011/06/2011060710345782_thumb.jpg)

这项研究由李德铢研究员领导张韵洁和马朋飞等人完成，李德铢研究员现任中国科学院昆明植物研究所所长，研究员，博士生导师。曾于1997年获国家杰出青年科学基金资助，同年入选中国科学院“百人计划”，2004年入选首批“新世纪百千万人才工程”国家级人选。 

竹亚科（Bambusoideae）隶属于禾本科（Poaceae），全世界共分布有一千余种。木本竹子因种类数目多，形态性状复杂多变及多年生一次性开花等原因而成为系统发育学研究难点。随着新一代测序技术的兴起，系统发育基因组学为解决这类困难类群的系统发育关系带来了曙光。 

中国科学院昆明植物研究所李德铢研究组对木本竹子的研究有较好的研究基础。最近，该研究组选取1种热带竹子和5种温带竹子，利用新一代测序技术 Illumina测定了这些种的叶绿体基因组全序列。通过与已知的2种竹子叶绿体基因组序列比对发现竹亚科植物的叶绿体基因组非常保守，具有良好的共线性和较低的分子进化速率，并证实了通过改良高盐低pH法直接分离叶绿体DNA并测序这一方法的可行性。 

基于24种禾本科植物叶绿体基因全序列，该研究组还构建了禾本科的系统发育树。在BEP分支中，竹亚科和早熟禾亚科（Pooideae）的姐妹群关系得到了强烈的支持，首次从系统发育基因组学的角度证实了近期的分子系统学研究结果，在竹亚科分支中热带分支和温带分支也分别为两个很好的单系。此外，与以前的研究结果相比，温带分支内部的系统学关系也得到了较好的解决。 

为探讨序列数据量的大小对系统关系的影响，研究组分别利用了四组数据分别构建系统树，分析表明，基于叶绿体基因组全序列的系统发育分析结果较为理想。对基因中插入或缺失的研究，发现45个插入或缺失中有25个可能为共近裔性状，20个可能为平行演化或回复突变的结果，且不同性质的插入或缺失在各个基因中的分布并不均匀，对其在系统发育学研究中的应用提供了参考。目前，采用叶绿体系统发育基因组学手段对温带竹子进行系统发育重建是比较行之有效的方法，这对于其它困难类群的系统学研究也有一定的启示意义。 

原文出处： 

PLoS ONE   DOI：10.1371/journal.pone.0020596 

High-Throughput Sequencing of Six Bamboo Chloroplast Genomes: Phylogenetic Implications for Temperate Woody Bamboos (Poaceae: Bambusoideae) 

Yun-Jie Zhang,Peng-Fei Ma,De-Zhu Li 

Background Bambusoideae is the only subfamily that contains woody members in the grass family, Poaceae. In phylogenetic analyses, Bambusoideae, Pooideae and Ehrhartoideae formed the BEP clade, yet the internal relationships of this clade are controversial. The distinctive life history (infrequent flowering and predominance of asexual reproduction) of woody bamboos makes them an interesting but taxonomically difficult group. Phylogenetic analyses based on large DNA fragments could only provide a moderate resolution of woody bamboo relationships, although a robust phylogenetic tree is needed to elucidate their evolutionary history. Phylogenomics is an alternative choice for resolving difficult phylogenies. Methodology/Principal Findings Here we present the complete nucleotide sequences of six woody bamboo chloroplast (cp) genomes using Illumina sequencing. These genomes are similar to those of other grasses and rather conservative in evolution. We constructed a phylogeny of Poaceae from 24 complete cp genomes including 21 grass species. Within the BEP clade, we found strong support for a sister relationship between Bambusoideae and Pooideae. In a substantial improvement over prior studies, all six nodes within Bambusoideae were supported with 鈮?.95 posterior probability from Bayesian inference and 5/6 nodes resolved with 100% bootstrap support in maximum parsimony and maximum likelihood analyses. We found that repeats in the cp genome could provide phylogenetic information, while caution is needed when using indels in phylogenetic analyses based on few selected genes. We also identified relatively rapidly evolving cp genome regions that have the potential to be used for further phylogenetic study in Bambusoideae. Conclusions/Significance The cp genome of Bambusoideae evolved slowly, and phylogenomics based on whole cp genome could be used to resolve major relationships within the subfamily. The difficulty in resolving the diversification among three clades of temperate woody bamboos, even with complete cp genome sequences, suggests that these lineages may have diverged very rapidly. 

 

<http://www.bioon.com/biology/postgenomics/486523.shtml>