title: Genome Biol：揭开袋鼠“跳着走”之谜
link: https://lufuhao.wordpress.com/2011/08/24/genome-biol%ef%bc%9a%e6%8f%ad%e5%bc%80%e8%a2%8b%e9%bc%a0%e8%b7%b3%e7%9d%80%e8%b5%b0%e4%b9%8b%e8%b0%9c/
author: lufuhao
description: 
post_id: 690
created: 2011/08/24 15:36:48
created_gmt: 2011/08/24 06:36:48
comment_status: open
post_name: genome-biol%ef%bc%9a%e6%8f%ad%e5%bc%80%e8%a2%8b%e9%bc%a0%e8%b7%b3%e7%9d%80%e8%b5%b0%e4%b9%8b%e8%b0%9c
status: publish
post_type: post

# Genome Biol：揭开袋鼠“跳着走”之谜

一些研究人员发现，袋鼠跳着走，基因是关键。

来自澳大利亚、美国、日本、英国和德国的科学家已完成塔马尔沙袋鼠的基因序列测定工作，最新研究成果报告刊载于《基因组生物学》（Genome Biology ）杂志。 

![2011082411494639](http://lufuhao.files.wordpress.com/2011/08/2011082411494639_thumb.jpg)

英国《每日电讯报》8月20日援引研究报告报道，研究人员在塔马尔沙袋鼠基因序列中发现一组基因，名为HOX。正是这一组基因使袋鼠拥有异常强壮的后肢，从而演化出极具特色的移动方式。 

其实，有袋动物和包括人类在内的有胎盘动物大约在1.8亿年前“是一家”，由同一个哺乳动物祖先分化进化而来。而塔马尔沙袋鼠是第三种接受基因序列测定的有袋动物。 

一些研究人员说，这次测序工作有利于了解整个哺乳动物的进化史。 

“观察一种与众不同的有机体是有价值的，可以了解人类和其他哺乳动物的进化过程，”有袋动物专家伊丽莎白·默奇森说：“借助观察在早期分化进化的哺乳动物，例如袋鼠，可以给你一个看待哺乳动物进化的新视角。” 

**Genome sequence of an Australian kangaroo, Macropus eugenii, provides insight into the evolution of mammalian reproduction and development**

Marilyn B Renfree, Anthony T Papenfuss, Janine E Deakin, James Lindsay, Thomas Heider, Katherine Belov, Willem Rens, Paul D Waters, Elizabeth A Pharo, Geoff Shaw, Emily SW Wong, Christophe M Lefevre, Kevin R Nicholas, Yoko Kuroki, Matthew J Wakefield, Kyall R Zenger, Chenwei Wang, Malcolm Ferguson-Smith, Frank W Nicholas, Danielle Hickford, Hongshi Yu, Kirsty R Short, Hannah V Siddle, Stephen R Frankenberg, Keng Yih Chew, Brandon R Menzies, Jessica M Stringer, Shunsuke Suzuki, Timothy A Hore, Ma 

Background We present the genome sequence of the tammar wallaby, Macropus eugenii, which is a member of the kangaroo family and the first representative of the iconic hopping mammals that symbolize Australia to be sequenced. The tammar has many unusual biological characteristics, including the longest period of embryonic diapause of any mammal, extremely synchronized seasonal breeding and prolonged and sophisticated lactation within a well-defined pouch. Like other marsupials, it gives birth to highly altricial young, and has a small number of very large chromosomes, making it a valuable model for genomics, reproduction and development. Results The genome has been sequenced to 2x coverage using Sanger sequencing, enhanced with additional next generation sequencing and the integration of extensive physical and linkage maps to build the genome assembly. We also sequenced the tammar transcriptome across many tissues and developmental time points. Our analyses of these data shed light on mammalian reproduction, development and genome evolution: there is innovation in reproductive and lactational genes, rapid evolution of germ cell genes, and incomplete, locus-specific X inactivation. We also observe novel retrotransposons and a highly rearranged major histocompatibility complex, with many class I genes located outside the complex. Novel microRNAs in the tammar HOX clusters uncover new potential mammalian HOX regulatory elements. Conclusions Analyses of these resources enhance our understanding of marsupial gene evolution, identify marsupial-specific conserved non-coding elements and critical genes across a range of biological systems, including reproduction, development and immunity, and provide new insight into marsupial and mammalian biology and genome evolution.