title: APPS: Extract high-quality plant RNA for NGS RNAseq
link: https://lufuhao.wordpress.com/2014/01/09/apps-extract-high-quality-plant-rna-for-ngs-rnaseq/
author: lufuhao
description: 
post_id: 1305
created: 2014/01/09 22:47:44
created_gmt: 2014/01/09 13:47:44
comment_status: open
post_name: apps-extract-high-quality-plant-rna-for-ngs-rnaseq
status: publish
post_type: post

# APPS: Extract high-quality plant RNA for NGS RNAseq

doi: [10.3732/apps.1300070](http://dx.doi.org/10.3732/apps.1300070)

###### A Method for Extracting High-Quality RNA from Diverse Plants for Next-Generation Sequencing and Gene Expression Analyses

Roxana Yockteng, Ana M. R. Almeida, Stephen Yee, Thiago Andre, Colin Hill, and Chelsea D. Specht 

  * _Premise_ _of_ _the_ _study:_ To study gene expression in plants, high-quality RNA must be extracted in quantities sufficient for subsequent cDNA library construction. Field-based collections are often limited in quantity and quality of tissue and are typically preserved in RNA_later._ Obtaining sufficient and high-quality yield from variously preserved samples is essential to studies of comparative biology. We present a protocol for the extraction of high-quality RNA from even the most recalcitrant plant tissues.

  * _Methods_ _and_ _Results:_ Tissues from mosses, cycads, and angiosperm floral organs and leaves were preserved in RNA_later_ or frozen fresh at −80°C. Extractions were performed and quality was measured for yield and purity.

  * _Conclusions:_ This protocol results in the extraction of high-quality RNA from a variety of plant tissues representing vascular and nonvascular plants. RNA was used for cDNA synthesis to generate libraries for next-generation sequencing and for expression studies using quantitative PCR (qPCR) and semiquantitative reverse transcription PCR (RT-PCR).

 

Materials and Methods

**Sampling** — Plant material was collected from greenhouses and botanical gardens ( Table 1 ) and either stored in RNA later (Ambion, Carlsbad, California, USA) or frozen immediately in liquid nitrogen.  reserved tissue was placed in long-term storage at −80 ° C. For storage in RNA later , approximately 3 × the volume of RNA later : tissue is used. Tissue stored in RNA later and frozen (−80 ° C) was defrosted just enough to remove the tissue from the RNA later prior to extraction. 

**Basic protocol** — The following protocol was modified from the manufacturer’s provided instructions for effective use of the Plant RNA Reagent from Life Technologies. As indicated, all solutions are prepared with sterile RNase-free water, and all supplies and handling materials are cleaned with RNase AWAY (Ambion) prior to dissection and storage. This protocol is optimized to isolate RNA from approximately 0.1 g of plant tissue. If the amount of plant tissue is increased, reagent volumes must be scaled appropriately. 

**Grinding the tissue**— One of the critical points to obtain high yield in the extraction of genetic material is the grinding. It is essential to grind the tissue as finely as possible, maintaining samples as cold as possible during grinding to avoid degradation. Selection of FastPrep or mortars/pestle depends on the hardness of the tissue being processed.  
A FastPrep FP120 Homogenizer (Thermo Savant, Carlsbad, California, USA) was used for grinding floral organs and soft leaf tissue. Approximately 0.1 g of frozen floral organs, whole flowers, and leaves or herbaceous stems were added to FastPrep 2-mL tubes (MP Biomedicals, Santa Ana, California, USA) 1/5 filled with bulk Lysing Matrix D (MP Biomedicals). FastPrep tubes containing the frozen tissue plus Lysing Matrix beads were shaken in the homogenizer (FastPrep) for 40 s at speed 6 (6 m/s) without buffer at room temperature.  
For grinding hard tissue or ligneous tissue, such as cone scales from gymnosperms, the tissue was ground under liquid nitrogen in a mortar and pestle that was sterilized and baked (minimum 12 h at 150 ° C). The tissue was ground as finely as possible, and the powdered material was placed in a 1.5-mL tube. The manually ground sample can be added to the FastPrep tube with Lysing Matrix beads (see above) for additional pulverization. 

**RNA extraction—** Once the tissue is sufficiently homogenized:  


>      1\. Add 0.6 mL of cold (4 °C) Plant RNA Reagent (Life Technologies) to pulverized tissue. Mix by brief vortexing or flicking the bottom of the tube until the sample is thoroughly resuspended. If tissue was ground with a FastPrep, homogenize with the cold buffer for an additional 40 s. If the tissue is not completely ground, repeat 1–2 × until the tissue is pulverized. If the tissue was ground with a mortar and pestle and does not need additional grinding, continue with the next step.  
     2\. Incubate 5 min at room temperature. Placing the samples on a rotator or nutator will help to maximize surface area of the tissue with the extraction buffer.  
      3\. Clarify the solution by centrifuging for 2 min at 12,000 × g in a microcentrifuge at room temperature. Transfer the supernatant to a tube with Phase Lock Gel (5 Prime, Gaithersburg, Maryland, USA). Although the Phase Lock Gel tubes are not required, they greatly facilitate separation of the organic and aqueous phases and help ensure cleanliness of the sample.  
      4\. Add 0.1 mL of 5 M NaCl to the sample, tap tube to mix, and add 0.3 mL of chloroform–isoamyl alcohol (24 : 1). Mix thoroughly by inversion and centrifuge the sample at 4 ° C for 10 min at 12,000 × g to separate the phases. Transfer the aqueous (top) phase to an RNase-free tube.  
      5\. Add to the aqueous phase an equal volume of a mix of LiCl (4 M) (3/4 v) and isopropyl alcohol (1/4 v). Mix and let stand at −20 ° C for 30 min to overnight. If the tissue was stored in RNA later , mix and let stand at −20 ° C for a maximum of 3 h (not overnight). If the precipitation is longer, salts from the RNA later solution could also precipitate. (Note: We have also let the sample stand at −80 ° C for 3 h, and this works as well.)  
      6\. Centrifuge the sample at 4 ° C for 20 min at 12,000 × g .  
      7\. Decant or remove supernatant with a pipette, taking care not to lose the pellet. Add 1 mL of 75% ethanol to the pellet. The pellet may be diffi cult to see. To help to see the pellet, you can add 1 μ L of GlycoBlue (Ambion) in step 5.  
      8\. Centrifuge at 4 ° C for 5 min at 12,000 × g . Decant the liquid carefully, taking care not to lose the pellet. Briefly centrifuge to collect the residual liquid, and remove it with a pipette.  
      9\. Let dry on ice for 15 min at room temperature and elute pellet in 10–30 μL of RNase-free water. Pipette the water up and down over the pellet to dissolve the RNA. If the pellet is difficult to dissolve, add more water or warm to 37 °C to facilitate the dissolution. It is important to resuspend the pellet completely to obtain an accurate measure of the concentration of your RNA. If the sample is not clean, it can be purified by the cleanup step suggested later. Although some protocols have suggested that performing an additional step of chloroform would clean the RNA samples (e.g., Accerbi et al., 2010 ), we found that an additional chloroform step decreases the RNA yield substantially.  


A NanoDrop ND-1000 spectrophotometer (Thermo Scientific, Wilmington, Delaware, USA) can be used to verify the concentration and purity of the RNA obtained. To assess the presence and purity of extracted RNA, use the ratio of absorbance at 230 nm, 260 nm, and 280 nm. If the RNA is pure, we expect a 260/280 ratio around 2, although this ratio does not guarantee pure RNA (see below). If the ratio is appreciably lower, it is an indication of the presence of protein, phenol, or other contaminants that absorb strongly at or near 280 nm. The ratio 260/230 is expected to be around 2–2.2. If this value is appreciably lower, it is an indication that contaminants such as carbohydrates, EDTA, guanidine isothiocyanate, and phenol that absorb at 230 nm are present in the sample. Ratios lower than expected could indicate that additional cleaning is necessary and the optional cleanup should be followed. While a more accurate assessment of the quality will be determined with a bioanalyzer prior to sequencing, this initial NanoDrop read will provide an indication of the presence of RNA, enabling the researcher to continue.

**** 

**Optional cleanup —** If the sample is not clean, the following modified cleanup procedure will help to purify the total RNA. This protocol is adapted from that published for DNA cleanup by Rohland and Reich (2012) , using magnetic beads to capture nucleotide material and permit additional washing steps that aid in the removal of undesirable metabolites. All stock solutions and reagents must be prepared with RNase-free water.  
      Preparing Sera-Mag beads solution— Begin by preparing 50 mL of SeraMag beads. Add 9 g of PEG-8000, 10 mL of 5 M NaCl, 500 μ L of 1 M Tris-HCl (pH 8.0), and 100 μ L of 0.5 M EDTA (pH 8.0) to a 50-mL Falcon tube. Add 1 mL (9 g) of the carboxyl-modified Sera-Mag Magnetic SpeedBeads (Thermo Scientific, Waltham, Massachusetts, USA; cat. no. 09-981-123) to a 1.5-mL tube. Pellet the beads for 5 min with a magnetic stand designed for 1.5–2-mL tubes. Remove the storage buffer, leaving the tubes in the magnetic stand. Wash beads 1 × with 1 mL TE or water. Remove the tube from the magnetic stand and immediately re-suspend the beads in an additional 1 mL TE or water.  
      Add the bead suspension to the prepared 50-mL Falcon tube and wrap the Falcon tube with aluminium foil. Store at 4 ° C for further use. The final concentration of the bead solution is 0.1% carboxyl-modified Sera-Mag Magnetic SpeedBeads, 18% PEG-8000 (w/v), 1 M NaCl, 10 mM Tris-HCl (pH 8.0), 1 mM EDTA (pH 8.0), and 0.05% Tween 20.

  
**RNA cleanup—** Bring a measured amount of prepared Sera-Mag beads solution to room temperature at least 30 min prior to use. For maximum binding, measure out 3 × the total RNA volume of the beads solution and place into a 1.5-mL tube along with the RNA. Place tubes on a rotator at room temperature for 10 min.  
      Spin the beads solution down before the beads dry permanently onto the wall of the tube. The beads must be spun down very quickly and at low speed to avoid bead precipitation; a mini benchtop centrifuge is recommended, with a spin so quick such that the lid does not need to be closed. Incubate without rotating at room temperature for an additional 10 min to maximize RNA binding. Place the 1.5-mL tubes in the magnetic stand for 5 min, then remove the buffer with a pipette and wash the beads 2 × with 500 μ L of fresh 80% ethanol. After the second wash, remove all ethanol and make sure that no ethanol is left in the samples. Remove the tubes from the magnetic stand and spin beads down from the sides of the wall before they dry. Air-dry the bead pellet for 10 min. Elute with RNase-free water using approximately the same volume as your original sample of RNA.  
      Vortex beads and water just to mix, and spin down quickly at low speed as before. Place tubes on a rotator at room temperature for 2–5 min for maximum RNA elution, and spin down quickly. Place tubes in the magnetic stand. Let stand for 5 min and recover eluted RNA in a separate tube.  
      A NanoDrop measure is recommended to check quality and yield of RNA.